src/controllers/v1/org-admin.js

```diff
@@ -144,4 +145,36 @@ module.exports = class OrgAdmin {
 			return error
 		}
 	}
+
+	/**
+	 * Bulk create session
+	 * @method
+	 * @name bulkSessionCreate
+	 * @param {String} req.body.file_path -Uploaded filr path .
+	 * @returns {Object} - uploaded file response.
+	 */
+	async bulkSessionCreate(req) {
+		try {
+			const sessionUploadRes = await orgAdminService.bulkSessionCreate(req.body.file_path, req.decodedToken)
+			return sessionUploadRes
+		} catch (error) {
+			return error
+		}
+	}
+
+	/**
+	 * Upload sample csv based on org id
+	 * @method
+	 * @name UploadSampleCsv
+	 * @param {String} req.body.file_path -Uploaded filr path .
+	 * @returns {Object} - uploaded file response.
+	 */
+	async bulkSessionCreate(req) {
```

Comment:
Why does this endpoint have the same name as above?

Reply:
@nevil-mathew I will remove it , i have missed it out . We need only one here